proto/quicksilver/interchainstaking/v1/proposals.proto

```diff
@@ -48,6 +49,7 @@ message RegisterZoneProposalWithDeposit {
   bool unbonding_enabled = 13 [(gogoproto.moretags) = "yaml:\"deposits_enabled\""];
   int64 decimals = 14 [(gogoproto.moretags) = "yaml:\"decimals\""];
   bool is_118 = 15;
+  int64 dustThreshold = 16 [(gogoproto.moretags) = "yaml:\"dust_threshold\""];
```

Comment:
will need to make this an sdk.Int() because the threshold for tokens with 18 decimals could overflow.